include/openssl/tls1.h

```diff
@@ -868,6 +872,9 @@ SSL_CTX_callback_ctrl(ssl,SSL_CTRL_SET_TLSEXT_TICKET_KEY_CB,(void (*)(void))cb)
 # define TLS1_TXT_DHE_PSK_WITH_CHACHA20_POLY1305           "DHE-PSK-CHACHA20-POLY1305"
 # define TLS1_TXT_RSA_PSK_WITH_CHACHA20_POLY1305           "RSA-PSK-CHACHA20-POLY1305"
 
+/* TLSv1.3 ciphersuites */
+# define TLS1_3_TXT_AES_128_GCM_SHA256                     "TLS_AES_128_GCM_SHA256"
```

Comment:
Selection of ciphers by name will get confused on duplicates. We went with AEAD-AES128-GCM-SHA256 which isn't great but avoids the collision with static RSA ciphers. (Whyyyy did they have to be named that way??? :-( )

Although we're probably also going to just disconnect the 1.3 ciphers from the old cipher language entirely. Otherwise everything which configures lists of ciphers will break in a world where 1.3 is on by default and the cipher selection in 1.3 involves far fewer complex tradeoffs between broken things, Cartesian products, etc.